src/main/java/DukeCommandValidator.java

```diff
@@ -0,0 +1,126 @@
+public class DukeCommandValidator {
+    static int MINIMUM_DONE_LENGTH = 5;
+    static int MINIMUN_TODO_LENGTH = 6;
+    static int MINIMUM_EVENT_LENGTH = 12;
+    static int MINIMUM_DEADLINE_LENGTH = 15;
```

Comment:
It's nice that all the values used are well-explained. Constants are also in uppercase and readable.